pegasus/sites.v3/code.org/public/css/maker-skinny-banner.scss

```diff
@@ -9,10 +9,10 @@
   position: relative;
 
   @media screen and (max-width: $width-md) {
-    padding: 0 1rem;
+    padding: 0 2rem;
   }
 
-  @media screen and (max-width: $width-sm) {
+  @media screen and (max-width: 546px) {
```

Comment:
Is `546px` a frequently used max-width? And/or could it be a constant that we use elsewhere?

Reply:
This banner is finicky because of the image bleed so I went custom on this one instead of using the `$width-sm` variable (which is `640px`) so it'll break cleaner at all sizes.

Reply:
I've found that banners work best with custom breakpoints, but other grid-based content is fine using the breakpoints found [here](https://github.com/code-dot-org/code-dot-org/blob/staging-next/shared/css/breakpoints.scss).

Reply:
Gotcha, thank you!